CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+Other improvements:
+- Remove run.js in preparation of `v0.15.0` release (#845)
```

Comment:
The current line might be cryptic to a future reader that doesn't know about `run.js` or `0.15.0`, so I propose we explain this with more context:
```suggestion
- Avoid writing a JS file when executing `spago run` (#845, #846)
```